docker-compose.yml

```diff
@@ -36,6 +36,27 @@ services:
     volumes:
       - ./frontend:/corpora-frontend
       - /corpora-frontend/node_modules/
+  processing:
+    image: corpora-processing
+    build:
+      context: .
+      dockerfile: Dockerfile.processing_image
+    restart: "no"
+    volumes:
+      - ./backend/corpora/dataset_processing:/code/dataset_processing
+      - ./backend/corpora/common:/code/common
+    command: ["sleep", "infinity"]
+    environment:
+      - PYTHONUNBUFFERED=1
+      - CORPORA_LOCAL_DEV=true
+      - AWS_REGION=us-west-2
+      - AWS_DEFAULT_REGION=us-west-2
+      - AWS_ACCESS_KEY_ID=dev_access_key_id
+      - AWS_SECRET_ACCESS_KEY=dev_secret_access_key
+      - BOTO_ENDPOINT_URL=http://localstack:4566
+      - DEPLOYMENT_STAGE=dev
+      - ARTIFACT_BUCKET=artifact-bucket
```

Comment:
Are these the correct bucket names? In s3 I'm seeing `sc-happy-corpora-artifact-dev` and `sc-happy-corpora-cellxgene-dev`